content/src/Server.ts

```diff
@@ -72,6 +72,7 @@ export class Server {
     this.registerRoute("/denylist/:type/:id", controller, controller.isTargetDenylisted, HttpMethod.HEAD);
     this.registerRoute("/failedDeployments", controller, controller.getFailedDeployments);
     this.registerRoute("/challenge", controller, controller.getChallenge);
+    this.registerRoute("/deltas/:type", controller, controller.getDeltas);
```

Comment:
Should we avoid the type as a path parameter? In the past, this kind of parameterization shown to be very limiting, and I think we may want to receive deltas from different kinds of types.